src/components/Meta.tsx

```diff
@@ -37,29 +43,26 @@ export default function render(
           "docs sectionType-" + props.sectionType + " " + props.category
         }
       />
+      {/* SEO content */}
+      <meta name="description" content={props.fields?.seoDescription || props.highlights}/>
+        
+      <meta property="og:image" content={ogImageURL} />
```

Comment:
We actually should have a default that should be overridable via the field in contentful, same for twitter